logstash-core.gemspec

```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |gem|
   gem.homepage      = "http://www.elastic.co/guide/en/logstash/current/index.html"
   gem.license       = "Apache License (2.0)"
 
-  gem.files         = Dir.glob(["lib/**/*.rb", "locales/*"])
+  gem.files         = Dir.glob(["lib/logstash-core.rb", "lib/logstash/**/*.rb", "spec/**/*.rb", "locales/*"])
```

Comment:
`logstash-event` is another issue we'll deal with eventually. it is currently a released gem still in use but in its current state we cannot generate a new version because I assume the its gemspec is very probably broken. we can deal with this in another issue. not urgent for now, but yes, we keep it in the repo for now.